functions-methods.js

```diff
@@ -21,6 +28,37 @@
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
 
+function typeOfEmail(emailAdress) {
+    const indexOfAap = emailAdress.indexOf("@");
+    const accountType = emailAdress.substring(indexOfAap + 1);
+
+
+    switch (accountType) {
+        case "novi-education.nl":
+            return "Student";
+            break;
```

Comment:
Als je een return gebruikt in een case, dan hoef je geen break te gebruiken, omdat die regel code nooit bereikt zal worden:
```suggestion
```